Makefile

```diff
@@ -34,24 +34,30 @@ build-java-components:
 	cd ./components; \
 	mvn clean install;
 
-.PHONY: build-sts-server-docker
-build-sts-server-docker:
-	cd ./docker/sts; \
-	bash build.sh;
-
 .PHONY: build-all
-build-all: build-java-components build-sts-server-docker
+build-all: build-java-components docker-push
```

Comment:
This builds everything except envoy oidc filter ? Don't we need to build that too ?